demos/usage-leaflet.py

```diff
@@ -0,0 +1,98 @@
+import dash
+from dash.dependencies import Input, Output
+import dash_html_components as html
+
+import dash_cytoscape as cyto
+cyto.load_extra_layouts()
+
+app = dash.Dash(__name__)
+server = app.server
+
+app.scripts.config.serve_locally = True
+app.css.config.serve_locally = True
```

Comment:
```suggestion
```
`serve_locally=True` is the default since dash 1.0, and anyway if you want `False` you can set it in the constructor `dash.Dash(serve_locally=False)`